src/pilot/deploy-overcloud.py

```diff
@@ -30,18 +36,534 @@
+                ssh_stdout_list, ssh_stderr_list = self.execute_cmd(sched_cmd)
+                time.sleep(2)
+
+                if ssh_stderr_list:
+                    logger.error("Failed to restart the scheduler service \
+                    with error".format(ssh_stderr_list))
+        except Exception as error:
+            message = "Exception {}: {}". \
+                      format(type(error).__name__, str(error))
+            logger.error("Failed to update filter on remote machine {} "
+                         "with error {}".format(self.ip_address, message))
+        finally:
+            # Close the ssh connection
+            self.ssh_connection.close()
+
+
+def edit_dell_environment_file(enable_hugepage,
+                               enable_numa,
+                               hugepage_size,
+                               vcpu_pin_set,
+                               dell_compute_count=0):
+    try:
+        logger.info("Editing dell environment file")
+        file_path = home_dir + '/pilot/templates/dell-environment.yaml'
+        if not os.path.isfile(file_path):
+            raise Exception("The dell-environment.yaml file does not exist")
+        cmds = ['sed -i "s|dellnfv::hugepages::enable:.*|' +
+                'dellnfv::hugepages::enable: ' +
+                str(enable_hugepage) +
+                '|" ' +
+                file_path,
+                'sed -i "s|dellnfv::numa::enable:.*|' +
+                'dellnfv::numa::enable: ' +
+                str(enable_numa) +
+                '|" ' +
+                file_path]
+
+        if dell_compute_count > 0:
+            cmds.append('sed -i "s|DellComputeCount:.*|' +
+                        'DellComputeCount: ' +
+                        str(dell_compute_count) +
+                        '|" ' +
+                        file_path)
+
+        if hugepage_size == "2MB":
+            hugepage_number = 49152
+        elif hugepage_size == "1GB":
+            hugepage_number = 96
+
+        if enable_hugepage:
+            cmds.append('sed -i "s|dellnfv::hugepages::hugepagesize:.*|' +
+                        'dellnfv::hugepages::hugepagesize: ' +
+                        hugepage_size[0:-1] +
+                        '|" ' +
+                        file_path)
+            cmds.append('sed -i "s|dellnfv::hugepages::hugepagecount:.*|' +
+                        'dellnfv::hugepages::hugepagecount: ' +
+                        str(hugepage_number) +
+                        '|" ' +
+                        file_path)
+        if enable_numa:
+            cmds.append('sed -i "s|dellnfv::numa::vcpu_pin_set:.*|' +
+                        'dellnfv::numa::vcpu_pin_set: \\\\\\\\\\"' +
+                        vcpu_pin_set +
+                        '\\\\\\\\\\"|" ' +
+                        file_path)
+
+        for cmd in cmds:
+            status = os.system(cmd)
+            logger.info("cmd: {}".format(cmd))
```

Comment:
The above logging line should be at the debug level.

Reply:
Changed to debug.